src/uk/gov/defra/ffc/ADO.groovy

```diff
@@ -1,25 +1,41 @@
 package uk.gov.defra.ffc
 
 class ADO implements Serializable {
-
   static void triggerPipeline(def ctx, String namespace, String chartName, String chartVersion, Boolean hasDatabase) {
-    if (hasDatabase) {
-      triggerDatabasePipeline(ctx, chartName, chartVersion)
+    String service = getServiceName(namespace)
+    if(service) {
+      if (hasDatabase) {
+        triggerDatabasePipeline(ctx, service, chartName, chartVersion)
+      }
+      triggerHelmPipeline(ctx, service, namespace, chartName, chartVersion)
     }
```

Comment:
Let's log that the pipeline(s) aren't going to be ran for `namespace` (or sim.)

Reply:
Agreed, done.